_posts/2025-05-20-firefox-security-privacy-newsletter-2025-q1.md

```diff
@@ -0,0 +1,72 @@
+
+**Mozilla Research Summit:** The [Mozilla Research Summit](https://surf.mozilla.org/events/2025/sandiego/) took place on March 1st, 2025 in San Diego California. We had 56 participants from 3 continents and 25+ different research institutions. The day included some high-caliber talks from Mozillians as well as security and privacy academics and sparked new research collaborations between Mozilla and the academic community.
+
+**Co-organized Open-Source Crypto Summit**: We co-organized the [Open Source Cryptography Workshop (OSCW)](https://opensourcecryptowork.shop/) which brings together practitioners interested in all sorts of open source crypto projects ranging from production and maintenance challenges to best practices for designing clean-sheet crypto systems with modern primitives.
+
+## Web Security & Standards
+
+**Fingerprinting:** We have taken on co-authorship of the [W3C’s guidance on Fingerprinting](https://w3c.github.io/fingerprinting-guidance/), to ensure that it matches our understanding of the evolving threads and the meaningful defenses we envision.  We’ve merged some changes already with more in discussion.
+
+**Messaging Layer Security**: Firefox is now the first browser to have a prototype implementation of MLS (Messaging Layer Security) ([Bug 1876002](https://bugzilla.mozilla.org/show_bug.cgi?id=1876002)). MLS enables end-to-end security for a wide range of use cases such as secure group messaging. MLS is available in Firefox 136 as an [origin trial](https://wiki.mozilla.org/Origin_Trials) (set the preference  dom.origin-trials.mls.state to 1\) which exposes an experimental Web API that will be used by our partners.
+
+**Sanitizer API:** We have [updated our implementation of the Sanitizer API](https://bugzilla.mozilla.org/show_bug.cgi?id=1956310), which allows to mitigate the risk of DOM-based cross-site scripting (XSS) attacks. This update aligns our implementation of the sanitizer API with recent developments and prepares it for a pre-release.
+
+**Integrity:** We presented our plans for wider Web App Integrity at the W3C WebAppSec working group ([explainer](https://github.com/beurdouche/explainers/blob/main/waict-explainer.md)) and started to work on [a first milestone towards script-integrity enforcements](https://github.com/w3c/webappsec-subresource-integrity/pull/133) as part of the Subresource Integrity specification with Yoav Weis from Shopify. We also [implemented integrity for import maps.](https://bugzilla.mozilla.org/show_bug.cgi?id=1945540)
```

Comment:
Maybe add a high-level sentence like "This will give web developers more tools to ensure their websites only runs the right code."